### DISCUSSION TITLE
Ease of Backlog Prioritization

### DISCUSSION THREAD

**Discussion by InternetMaster1** [community/community#4686] [Projects and Issues]
I love the project tables and custom fields. 

To make backlog prioritization easier, 
1) Are there quick options to reorder the rows using drag and drop? 

2) Is there advanced sorting similar to Excel? I would like to sort by priority, then by Assignee, and then by a custom field.

The above two features would make backlog planning a breeze for the product owner.

Thanks!

**Comment by 0xdevalias** [comment]
> Are there quick options to reorder the rows using drag and drop?

From a quick check, it seems like I can drag/drop in the 'table' view.

> Is there advanced sorting similar to Excel?

From a quick check on the 'table' view, it looked like I could only sort by 1 column at a time (but I didn't look very hard to see if there was another way to sort by multiple)

**Comment by InternetMaster1** [comment]
It's lovely to know that drag and drop is available..

It would be nice to have multiple sorting feature.

**Comment by zepfietje** [comment]
There's now a dedicated discussion for multiple sort columns: https://github.com/github/feedback/discussions/6262.

Would be great if you could mark this discussion as answered and upvote the other.
